dev-ml/camomile/files/ocaml406.patch

```diff
@@ -0,0 +1,13 @@
+--- camomile-0.8.5/Makefile.in  2013-09-15 02:48:38.000000000 -0500
++++ camomile-0.8.5.new/Makefile.in      2020-07-03 00:43:13.639039206 -0500
+@@ -48,8 +48,8 @@
+ OCAMLMKLIB = @OCAMLMKLIB@
+
+ # compiler options
+-BOPTIONS = @DEBUG@ @ASSERT@
+-OOPTIONS = @COPTIONS@ @PROFILE@ @ASSERT@
++BOPTIONS =@DEBUG@ @ASSERT@ -unsafe-string
++OOPTIONS =@COPTIONS@ @PROFILE@ @ASSERT@ -unsafe-string
```

Comment:
Are these options available only on certain OCaml versions? It'd be nicer to apply unconditionally but I appreciate that may not be possible.

Reply:
You've got me there. I think maaaaybe the deal is you didn't *need* to specify -unsafe-string until recent OCaml hardened strings by default? I'm reaching.

Reply:
Brilliant, thanks!